ui/src/components/DeleteModal/DeleteModal.test.tsx

```diff
@@ -0,0 +1,65 @@
+import { render, screen } from '@testing-library/react';
+import React, { createContext } from 'react';
+import userEvent from '@testing-library/user-event';
+import axios from 'axios';
+import DeleteModal from './DeleteModal';
+
+jest.mock('immutability-helper');
+jest.mock('../../util/util');
+jest.mock('axios');
+
+const handleClose = jest.fn();
+
+const TableContext = createContext({
+    rowData: { serviceName: { stanzaName: 1 } },
+    setRowData: () => {},
+});
+
+beforeEach(() => {
+    render(
+        <TableContext.Provider
+            value={{ rowData: { serviceName: { stanzaName: 1 } }, setRowData: () => {} }}
+        >
+            <DeleteModal
+                handleRequestClose={handleClose}
+                serviceName="serviceName"
+                stanzaName="stanzaName"
+                page="inputs"
+                open
+            />
+        </TableContext.Provider>
+    );
+});
+
+it('should render delete modal correctly', () => {
+    const deleteModal = screen.getByTestId('modal');
+    expect(deleteModal).toBeInTheDocument();
+
+    const cancelButton = screen.getByRole('button', { name: /cancel/i });
+    expect(cancelButton).toBeInTheDocument();
+
+    const deleteButton = screen.getByRole('button', { name: /delete/i });
+    expect(deleteButton).toBeInTheDocument();
+});
+
+it('close model and callback after cancel click', async () => {
+    const cancelButton = screen.getByRole('button', { name: /cancel/i });
+
+    await userEvent.click(cancelButton);
+    expect(handleClose).toHaveBeenCalled();
+});
+
+it('correct delete request', async () => {
+    (axios as unknown as jest.Mock).mockResolvedValue(
```

Comment:
I meant
```suggestion
    server.use(
        rest.delete('/servicesNS/nobody/-/mockGeneratedEndPointUrl', async (req, res, ctx) =>
            res(ctx.status(201))
        )
    );
```

and 

```ts
const deleteHandlerException = rest.delete(
    '/servicesNS/nobody/-/mockGeneratedEndPointUrl',
    async (req, res, ctx) =>
        res(ctx.status(500), ctx.json({ message: 'Deliberately broken request' }))
);

server.use(deleteHandlerException);
```
for assert to show error

Reply:
tip: you don't need to mock utils in this case

Reply:
if i do not mock the utils I will need to add the unified config mockup data(after removal it throws error that config is empty), but they are not used directly here, so seems like no need for it